cmake/modules/OmrUtility.cmake

```diff
@@ -118,7 +117,7 @@ function(omr_process_template input output)
 	file(READ "${input_abs}" template)
 	string(CONFIGURE "${template}" configured_template ${configure_args})
 	file(GENERATE OUTPUT "${output}" CONTENT "${configured_template}")
-endfunction()
+endfunction(omr_process_template)
```

Comment:
I'm not 100% sure but I believe we have been moving away from putting the function name in the closing `endfunction()`/`endmacro()`s. @rwy0717 ?

Reply:
I was following the lead of most of the other functions in this file.

Reply:
Yes we are, but as keith says, the majority of functions in this file have it, so I think this is fine.